scripts/release.sh

```diff
@@ -361,11 +417,26 @@ function parse_flags() {
             ;;
           --release-gcr)
             KO_DOCKER_REPO=$1
-            has_gcr_flag=1
+            has_registry_flag=1
+            ;;
+          --release-acr)
+            AZ_ACR_NAME=$1
+            KO_DOCKER_REPO="${1}.azurecr.io"
+            has_registry_flag=1
+            ;;
+          --release-azblob)
+            RELEASE_AZ_BLOB_URL=$1
+            has_storage_flag=1
             ;;
           --release-gcs)
             RELEASE_GCS_BUCKET=$1
-            has_gcs_flag=1
+            has_storage_flag=1
+            ;;
+          --validation-tests)
+            VALIDATION_TEST=$1
+            ;;
+          --test-args)
+            VALIDATION_TEST_ARGS=$1
```

Comment:
Quote, or you'll loose the ability to pass multiple arguments.

```suggestion
            VALIDATION_TEST_ARGS="$1"
```